relay-event-normalization/src/event.rs

```diff
@@ -455,6 +458,18 @@ fn normalize_dist(distribution: &mut Annotated<String>) {
     });
 }
 
+/// Defaults required attributes to their identity value.
+fn normalize_default_required_attrs(event: &mut Event) {
+    // The defaulting behavior, was inherited from `StoreNormalizeProcessor` and it's put here since only light
+    // normalization happens before metrics extraction and we want the metrics extraction pipeline to already work
+    // on some normalized data.
+    event.platform.get_or_insert_with(|| "other".to_string());
+    event.level.get_or_insert_with(|| match event.ty.value() {
```

Comment:
I have moved both `platform` and `level` normalization, since both defaulted to values which might be useful for metrics extraction. I could technically move all the default normalizations, @iker-barriocanal what do you think?

Reply:
This is good enough for now, thanks!